packages/Checkbox/src/checkbox.scss

```diff
@@ -1,291 +1,367 @@
 }
 
 .ids-form-control {
-  display: inline-flex;
-  align-items: flex-start;
+    display: inline-flex;
+    align-items: flex-start;
 }
 
 .ids-checkbox__label {
-  font-family: var(--ids-checkbox-font-family);
-  font-size: var(--ids-checkbox-font-size);
-  font-weight: var(--ids-checkbox-font-weight);
-  color: var(--ids-checkbox-text);
-
-  &--completion {
-    align-items: center;
-    display: flex;
-  }
+    font-family: var(--ids-checkbox-font-family);
+    font-size: var(--ids-checkbox-font-size);
+    font-weight: var(--ids-checkbox-font-weight);
+    color: var(--ids-checkbox-text);
+
+    &--completion {
+        align-items: center;
+        display: flex;
+    }
 }
 
 .ids-checkbox__label-text {
-  padding-left: var(--ids-checkbox-margin);
+    padding-left: var(--ids-checkbox-margin);
 }
 
 .ids-checkbox {
-  appearance: none;
-  height: var(--ids-checkbox-size);
-  width: var(--ids-checkbox-size);
-  flex-shrink: 0;
-  margin: .2rem 0 0;
-  display: inline-block;
-  position: relative;
-  cursor: pointer;
-  color: var(--ids-checkbox-check);
-  border: .1rem solid var(--ids-checkbox-border);
-  border-radius: var(--ids-checkbox-radius);
-  background: var(--ids-checkbox-background);
-  box-sizing: content-box;
-  transition: background 0.05s ease-in-out;
-
-  &:checked {
-    border-color: var(--ids-checkbox-border-checked);
-    background: var(--ids-checkbox-background-checked);
-  }
-
-  &:indeterminate {
-    border-color: var(--ids-checkbox-border-checked);
-    background: var(--ids-checkbox-background-checked);
-  }
-
-  &:disabled {
-    cursor: not-allowed;
-    border-color: var(--ids-checkbox-border);
-    background: var(--ids-checkbox-background-disabled);
-  }
-
-  &:focus {
-    outline: none;
-    box-shadow: var(--ids-checkbox-focus);
-  }
-
-  &:hover,
-  &:checked:hover,
-  &:indeterminate:hover {
-    border-color: var(--ids-checkbox-border-hover);
-  }
-
-  &:focus:not(:focus-visible) {
-    box-shadow: none;
-  }
-
-  &:checked:disabled,
-  &:indeterminate:disabled {
-    background: var(--ids-checkbox-background-checked-disabled);
-  }
-
-  &:checked::after,
-  &:indeterminate::after {
-    content: '';
-    position: absolute;
-  }
-
-  &:checked::after {
-    @extend %check;
-
-    display: block;
-    left: .2rem;
-    top: .7rem;
-    animation: check-animation ease-out 0.25s 0.15s both;
-  }
-
-  &:indeterminate::after {
-    @extend %indeterminate;
-
-    left: 50%;
-    top: 50%;
-    transform: translate(-50%, -50%);
-  }
-
-  &:checked:disabled::after {
-    animation: none;
-  }
-
-  &:disabled:hover {
-    border-color: var(--ids-checkbox-border);
-  }
+    appearance: none;
+    height: var(--ids-checkbox-size);
+    width: var(--ids-checkbox-size);
+    flex-shrink: 0;
+    margin: var(--ids-checkbox-top-space) 0 0;
+    display: inline-block;
+    position: relative;
+    cursor: pointer;
+    color: var(--ids-checkbox-check);
+    border: .1rem solid var(--ids-checkbox-border);
```

Comment:
this won't compute the same between both themes as the rem is not on the same base.